test/unit/db_tests.js

```diff
@@ -36,7 +36,7 @@ describe('Database', function() {
     test: function() {
       sinon.stub(test.utils, 'executeOperation').callsFake((topology, operation, args) => {
         const options = args[args.length - 2];
-        expect(options.readPreference).to.equal('primary');
+        expect(options.readPreference.equals(ReadPreference.primary)).to.be.ok;
```

Comment:
should we look into adding custom chai matchers for our objects like `ReadPreference`, `ClientSession`, etc?

Reply:
sure!